index.js

```diff
@@ -184,12 +184,11 @@ FCPClient.prototype.postConfigFiles = function (uniminifiedJSStr, minifiedJSStr,
 };
 
 /**
- * Post new config JS files
- * @param uniminifiedJSStr {String} String containing the unminified JS file
- * @param minifiedJSStr {String} String containing the minified JS file
+ * Post new code JS files
+ * @param codeBuffer {Buffer} The zipped code
  * @param notes {String} Comments on this release
  * @param version {String} Semver version
- * @param latest {Bool} Is this the latest?
+ * @param latest {Bool|String} Is this the latest? (or invalid?)
```

Comment:
really not a fan of overloading this param, but because of callback last signature we are almost stuck... ugh

Reply:
yeah I am not a fan either, but it has to be backwards compatible :-/